### PR TITLE
feat: Adding provision to have a result processor for workers

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description

This PR adds a feature to allow for generic post-processing of results.

Coped from: https://github.com/camunda-community-hub/spring-zeebe/pull/833

### CODE REVIEW

**Review by nicpuppa — APPROVED** [review]
LGTM 👍

File: spring-boot-starter-camunda-sdk/CONTRIBUTORS.md [hunk 1]
```diff
@@ -59,4 +59,4 @@
 * [minglei.wen](https://github.com/wenminglei)
 * [pcln-jaynil-patel](https://github.com/pcln-jaynil-patel)
 * [rob2universe](https://github.com/rob2universe)
-
+* [hem-verma](https://github.com/hem-verma)
```
Comment (nicpuppa) [inline spring-boot-starter-camunda-sdk/CONTRIBUTORS.md:62]: Hey @jonathanlukas, the Java Checkstyle CI is complaining about this file, can you add an empty line at the end ?

Comment (jonathanlukas) [reply spring-boot-starter-camunda-sdk/CONTRIBUTORS.md:62]: There is an empty line at the end.

Comment (jonathanlukas) [reply spring-boot-starter-camunda-sdk/CONTRIBUTORS.md:62]: I let the formatter run. It somehow requires 2 empty lines...

**Pull request merged**